<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAreaMenu</class>
 <widget class="QDialog" name="CAreaMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>807</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="headline">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>491</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Area-Menü</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="CMyView" name="areaView">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>651</width>
     <height>351</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="setPB">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>710</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Zielland auswählen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitPB">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>710</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Schließen</string>
   </property>
  </widget>
  <widget class="QSplitter" name="unitsInAreaLayout">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>490</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="unitsInAreaLabel">
    <property name="text">
     <string>Einheiten in Area: </string>
    </property>
   </widget>
   <widget class="QLabel" name="unitsInAreaInfo">
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="availableUnitsLayout">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>580</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="availableUnitsLabel">
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Verfügbare Einheiten in Area:</string>
    </property>
   </widget>
   <widget class="QLabel" name="availableUnitsInfo">
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="supplyRatioLayout">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>490</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="supplyRatioLabel">
    <property name="text">
     <string>Versorgunsrate:</string>
    </property>
   </widget>
   <widget class="QLabel" name="supplyRatioInfo">
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="mayAddUnitsLayout">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>530</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="maxAddUnitsLabel">
    <property name="text">
     <string>Maximal noch mögliche Einheiten innerhalb des Versorgungslimits:</string>
    </property>
   </widget>
   <widget class="QLabel" name="maxAddUnitsInfo">
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="setUnitsLayout">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>640</y>
     <width>251</width>
     <height>35</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="setUnitsLabel">
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Einheiten setzen:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="setUnitsSB">
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="areaSizeLayout">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>490</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="areaSizeLabel">
    <property name="text">
     <string>Anzahl Länder:</string>
    </property>
   </widget>
   <widget class="QLabel" name="areaSizeInfo">
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="maxValuePB">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>640</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>max.</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CMyView</class>
   <extends>QGraphicsView</extends>
   <header>myview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
